src/components/multiselect.vue

```diff
@@ -49,13 +49,16 @@ except according to the terms contained in the LICENSE file.
           :class="{ 'shows-all': searchValue === '' }" @change="changeCheckbox">
           <template v-if="options != null">
             <!-- eslint-disable-next-line vue/object-curly-newline -->
-            <li v-for="({ value, key = value, text = value, title = text }, i) in options"
+            <li v-for="({ value, key = value, text = value, description }, i) in options"
               :key="key" :class="{ 'search-match': searchMatches.has(value) }">
               <div class="checkbox">
                 <label>
-                  <input type="checkbox" :data-index="i">
-                  <span :title="title">{{ text }}</span>
+                  <input type="checkbox" :data-index="i"
+                    :aria-describedby="description != null ? descriptionId(i) : null">
+                  <span v-if="description == null" v-tooltip.text>{{ text }}</span>
+                  <span v-else v-tooltip.no-aria="description">{{ text }}</span>
```

Comment:
This is `v-tooltip.no-aria`, but the description will be surfaced to screen readers using the `aria-describedby` attribute above. This is an example of how `v-tooltip.no-aria` can be used in complex/rare situations.